core/src/utils/test/overlays/overlays.spec.ts

```diff
@@ -5,127 +5,194 @@ import { Nav } from '../../../components/nav/nav';
 import { RouterOutlet } from '../../../components/router-outlet/router-outlet';
 import { setRootAriaHidden } from '../../overlays';
 
-describe('setRootAriaHidden()', () => {
-  it('should correctly remove and re-add router outlet from accessibility tree', async () => {
-    const page = await newSpecPage({
-      components: [RouterOutlet],
-      html: `
-        <ion-router-outlet></ion-router-outlet>
-      `,
+describe('overlays', () => {
```

Comment:
The first `describe` block is unchanged; the second one contains my new tests. I wrapped both in a single parent `describe` as a workaround for a Stencil bug discussed here: https://outsystems.slack.com/archives/C06BR3RGLHM/p1707342287102049 We use this pattern in plenty of other spec tests, so I don't see this needing a comment or tech debt ticket, but LMK if you'd like me to add one.